Description: No require rubygems
Subject: Do not require rubygems
 We do not require rubygems; the apt repository should contain all
 dependencies.
Forwarded: not-needed
From: Stig Sandbeck Mathisen <ssm@debian.org>
--- a/bin/facter
+++ b/bin/facter
@@ -58,17 +58,6 @@
 # Copyright (c) 2011 Puppet Labs, Inc
 # Licensed under the Apache 2.0 license
 
-# Bundler and rubygems maintain a set of directories from which to
-# load gems. If Bundler is loaded, let it determine what can be
-# loaded. If it's not loaded, then use rubygems. But do this before
-# loading any facter code, so that our gem loading system is sane.
-if not defined? ::Bundler
-  begin
-    require 'rubygems'
-  rescue LoadError
-  end
-end
-
 require 'facter/application'
 
 Facter::Application.run(ARGV)
